azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/models/MessageInfoModel.kt

```diff
@@ -15,6 +15,7 @@ internal data class MessageInfoModel(
     val internalId: String? = null,
     val messageType: ChatMessageType?,
     val content: String?,
+    val topic: String? = null,
```

Comment:
We might want to copy the topic when we merge messages in the list & tree based implementation. Could you add that with a commit with this PR?

fun mergeWithPreviousMessage() 
in this function for both ListStorage & TreeStorage classes.

Reply:
Topic/Participants won't ever get hit with a EditMessage, so shouldn't ever change after being created.